### PR TITLE
feat: migrate landing panel to Mantine

### PR DESCRIPTION
**Pull request by yasmin-e**
Closes: #4810 

### Description:
before
Screenshot 2023-03-16 at 15 39 11

after
Screenshot 2023-03-17 at 12 37 27

### CODE REVIEW

**Comment by hamzahc1** [comment]
@yasmin-e - the Run a Query icon is different to previously, is this intentional?

**Comment by yasmin-e** [comment]
> @yasmin-e - the Run a Query icon is different to previously, is this intentional?

yes! assume that all icon changes on mantine migrations are prev. approved by @PriPatel  :)

**Comment by PriPatel** [comment]
Yes! 
For a few reasons

1. The old icon doesnt make much sense and we couldn't find a suitable match
2. The table icon is more consistent with what you see in the navbar
Screenshot 2023-03-16 at 19 15 04

The other option was no icon - Im also happy with that!

File: packages/frontend/src/components/common/PrimaryLinkButton.tsx
Comment (IrakliJani) [inline packages/frontend/src/components/common/PrimaryLinkButton.tsx:1]: ignoring

File: packages/frontend/src/components/Home/OnboardingPanel/index.tsx
Comment (IrakliJani) [inline packages/frontend/src/components/Home/OnboardingPanel/index.tsx:1]: ignoring

File: packages/frontend/src/components/Home/OnboardingPanel/OnboardingSteps.styles.tsx
Comment (IrakliJani) [inline packages/frontend/src/components/Home/OnboardingPanel/OnboardingSteps.styles.tsx:1]: ignoring

File: packages/frontend/src/components/Home/LandingPanel/LandingPanel.styles.tsx [hunk 1]
```diff
@@ -1,26 +1,6 @@
...
-    color: ${Colors.BLACK};
-`;
-
-export const Intro = styled.p`
-    color: ${Colors.GRAY1};
-`;
-
 export const StyledLinkButton = styled(LinkButton)`
     font-size: 14px !important;
 `;
```
Comment (IrakliJani) [inline packages/frontend/src/components/Home/LandingPanel/LandingPanel.styles.tsx:6]: are we using this someplace else? if yes, can we get rid of it from this file and move where it's necessary?

**Review by IrakliJani — CHANGES_REQUESTED** [review]
minor comments

**Comment by IrakliJani** [comment]
re-running tests ✅

**Review by IrakliJani — APPROVED** [review]

**Pull request merged**